hardhat.config.js

```diff
@@ -32,36 +32,36 @@ module.exports = {
     cache: "build/cache",
     deploy: "deploy",
     sources: "contracts",
-    imports: 'imports'
+    imports: "imports",
   },
   defaultNetwork: "hardhat",
   networks: {
     localhost: {
       ...sharedNetworkConfig,
       blockGasLimit: 100000000,
       gas: 2000000,
-      saveDeployments: true
+      saveDeployments: false,
     },
     hardhat: {
       blockGasLimit: 100000000,
       gas: 2000000,
-      saveDeployments: true
+      saveDeployments: false,
```

Comment:
Just for my own understanding, what does this variable do? I couldn't find it by looking into the hardhat config doc

Reply:
The flag determines whether fresh deployments are saved in the `deployments` folder. We don't need this for the hardhat netwwork, because it means that every time we run our tests (and contracts are deployed to the hardhat netowrk for testing) the deployment information are saved to the deployments folder, which is not really necessary.

Reply:
https://hardhat.org/plugins/hardhat-deploy.html#savedeployments